source/manual/development-pipeline.html.md

```diff
@@ -14,83 +14,135 @@ review_in: 6 months
 Our development and deployment pipeline looks like this:
 
 1. Open a Pull Request (PR)
-1. [Wait for Continuous Integration to pass](#wait-for-continuous-integration-to-pass)
+1. [Wait for automated checks to pass](#wait-for-automated-checks-to-pass)
 1. [Get someone to review your Pull Request](#get-someone-to-review-your-pull-request)
 1. [Check if there is a deploy freeze](#check-if-there-is-a-deploy-freeze)
 1. [Merge your own Pull Request](#merge-your-own-pull-request)
-1. [Wait for the release to deploy to Integration](#wait-for-the-release-to-deploy-to-integration)
-1. [Manually deploy to Staging, then Production](#manually-deploy-to-staging-then-production)
+1. [Deploy through each of the environments](#deployment)
 
-## Wait for Continuous Integration to pass
+## Wait for automated checks to pass
 
-When a Pull Request (PR) is opened, a job starts running on [our CI infrastructure](/manual/test-and-build-a-project-on-jenkins-ci.html). In most cases the CI job will run 3 different checks on the repo: a [Ruby linter](https://github.com/alphagov/rubocop-govuk), a [static security analysis tool](/manual/brakeman.html), and the [unit tests of the repo](/manual/testing.html).
+When a Pull Request (PR) is opened, it often triggers an automated
+job, which typically lints the code and runs the tests.
 
-> For some applications, Jenkins also runs the [end-to-end tests](/manual/publishing-e2e-tests.html).
+Increasingly we're using [GitHub Actions](/manual/test-and-build-a-project-with-github-actions.html)
+to perform these checks, but historically we've run these jobs on
+[our CI infrastructure](/manual/test-and-build-a-project-on-jenkins-ci.html).
+Some applications also trigger a run of the [end-to-end tests](/manual/publishing-e2e-tests.html).
 
 ## Get someone to review your Pull Request
 
 The GDS Way has guidelines on [how to review code](https://gds-way.cloudapps.digital/manuals/code-review-guidelines.html). Only when the tests pass and the code has been approved the Pull Request can be merged, since we've [configured GitHub to prevent merges](/manual/configure-github-repo.html) otherwise.
 
+Other than code changes, reviewers may refer to the following:
```

Comment:
This makes it sound like it's the reviewer's job to do this, but it could well be the implementer. Can we make this less specific?

Reply:
Agreed - I've moved it to its own section, 'Review your own changes'.